server/src/main/java/com/objectcomputing/checkins/gcp/postgres/GoogleCloudDatabaseSetup.java

```diff
@@ -0,0 +1,34 @@
+package com.objectcomputing.checkins.gcp.postgres;
+
+import javax.inject.Singleton;
+
+import io.micronaut.configuration.jdbc.hikari.DatasourceConfiguration;
+import io.micronaut.context.annotation.Property;
+import io.micronaut.context.annotation.Requires;
+import io.micronaut.context.env.Environment;
+import io.micronaut.context.event.BeanCreatedEvent;
+import io.micronaut.context.event.BeanCreatedEventListener;
+
+@Singleton
+@Requires(env = Environment.GOOGLE_COMPUTE)
```

Comment:
@Maroju100 Here's your answer to limiting the basic auth to tests: Environment.TEST